README.md

```diff
@@ -368,294 +396,260 @@ A device renderer instance can be configured using the `options` argument (objec
 -   **Details:**
-    Enables or disables the rotation widget. This widget can be used to rotate the Android virtual device.
-
-### `navbar`
+    Enables or disables the fileUpload widget and drag & drop. This widget can be used to forward local file to the Android virtual device. When drag & dropping APK or ZIP files, it will install them.
 
-<img align="right" src="./doc/assets/ic_nav_android_back.svg" alt="..."></img>
-<img align="right" src="./doc/assets/ic_nav_android_home.svg" alt="..."></img>
-<img align="right" src="./doc/assets/ic_nav_android_multiapp.svg" alt="..."></img>
+#### `fileUploadUrl`
 
--   **Type:** `Boolean`
--   **Default:** `true`
+-   **Type:** `String`
+-   **Default:** `undefined`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the navbar widgets. This widget can be used to navigate in the Android virtual device like when using hardware buttons.
-
-### `power`
+    Set the file upload URL, required if `fileUpload` is set to `true`.
 
-<img align="right" src="./doc/assets/ic_power.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_fullscreen.svg"> `fullscreen`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Fullscreen`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the power widget. This widget can be used to poweroff or reboot the Android virtual device.
-
-### `fullscreen`
+    Enables or disables the fullscreen widget. This widget can be used to make the renderer go fullscreen.
 
-<img align="right" src="./doc/assets/ic_fullscreen_active_black.svg" alt="..."></img>
-<img align="right" src="./doc/assets/ic_fullscreen_exit_active_black.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_gamepad.svg"> `gamepad`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
--   **Compatibility:** `PaaS`, `SaaS`
+-   **Toolbar name:** `Gamepad`
+-   **Compatibility:** `SaaS`, `PaaS`
 -   **Details:**
-    Enables or disables the fullscreen widget. This widget can be used to make the renderer go fullscreen.
+    Enable or disable gamepad support & widget.
 
-### `camera`
-
-<img align="right" src="./doc/assets/ic_camera.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_location.svg"> `gps`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `GPS`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the camera widget. This widget can be used to forward local webcam video to the Android virtual device.
-    By default, if the `microphone` property is also true, then the default audio input will be used as well.
+    Enables or disables the GPS widget. This widget can be used to set the GPS location of the Android virtual device. If you want to use a visual map instead of GPS coordinates number to set the location, you must import the Google Maps library with your API key.
```

Comment:
```suggestion
    Enables or disables the GPS widget. This widget can be used to set the GPS location of the Android virtual device. If you want to use a visual map instead of GPS coordinates to set the location, you must import the Google Maps library with your API key.
```